CrossPare/src/main/java/de/ugoe/cs/cpdp/execution/AbstractCrossProjectExperiment.java

```diff
@@ -181,27 +264,28 @@ public void run() {
                     continue;
                 }
 
-                // Setup testdata and training data
-                Instances testdata = testVersion.getInstances();
-                List<Double> efforts = testVersion.getEfforts();
-                List<Double> numBugs = testVersion.getNumBugs();
-                Instances bugMatrix = testVersion.getBugMatrix();
-                SetUniqueList<Instances> traindataSet =
-                    SetUniqueList.setUniqueList(new LinkedList<Instances>());
+                // Setup testdata and training data                
+                SoftwareVersion testversion = new SoftwareVersion(testVersion);
+                Instances testdata = testversion.getInstances();
+                List<Double> efforts = testversion.getEfforts();
+                List<Double> numBugs = testversion.getNumBugs();
+                Instances bugMatrix = testversion.getBugMatrix();

```

Comment:
Are these still needed? You can just get them from the testVersion, right?

Reply:
+1